src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_download_linux_web_log.yaml

```diff
@@ -63,24 +13,25 @@ interactions:
       ParameterSetName:
       - -g -n --is-linux
       User-Agent:
-      - python/3.6.6 (Windows-10-10.0.18362-SP0) msrest/0.6.6 msrest_azure/0.6.0 resourcemanagementclient/2.1.0
-        Azure-SDK-For-Python AZURECLI/2.0.66
+      - python/2.7.15+ (Linux-4.4.0-18362-Microsoft-x86_64-with-Ubuntu-18.04-bionic)
+        msrest/0.6.9 msrest_azure/0.6.1 azure-mgmt-resource/3.1.0 Azure-SDK-For-Python
+        AZURECLI/2.0.71
       accept-language:
       - en-US
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2018-05-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2019-05-10
   response:
     body:
-      string: '{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001","name":"clitest.rg000001","location":"westus","tags":{"product":"azurecli","cause":"automation","date":"2019-06-07T22:23:06Z"},"properties":{"provisioningState":"Succeeded"}}'
+      string: !!python/unicode '{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001","name":"clitest.rg000001","type":"Microsoft.Resources/resourceGroups","location":"westus","tags":{"date":"2019-08-19T21:00:23Z","product":"azurecli","cause":"automation"},"properties":{"provisioningState":"Succeeded"}}'
```

Comment:
Because the `!!python/unicode` in response body, response body cannot be parsed correctly. Why do you have such string in your recording file?  When I delete `!!python/unicode` manually, it works. In addition, please have a look at other `!!python/unicode` in the following part.